understanding/contrast-enhanced.html

```diff
@@ -135,223 +130,113 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Text can be seen by people who need strong contrast.</dd>
+                  <dt>目標</dt>
+                  <dd>テキストを、強いコントラストが必要な人にも見られるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Strongly contrast text against its background.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>背景に対してテキストを強くコントラストさせる。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people cannot read text with minimum contrast.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>最低限のコントラストのテキストを読むことができない人がいる。</dd> 
```

Comment:
これちょっとわかりにくいですね。最低限のコントラスト基準を満たしたテキストであってもなお読めない人がいる、という話なのですが。

原文もわかりにくいのでママで良いかとも思いつつ、多少補って以下のようにしてみるのもありかもしれません。ここはご意見をお聞きしたいです。

```suggestion
                  <dd>最低限のコントラストを保ったテキストでも、読むことができない人がいる。</dd> 
```